src/aiortc/contrib/media.py

```diff
@@ -153,10 +155,23 @@ def player_worker(
             if video_first_pts is None:
                 video_first_pts = frame.pts
             frame.pts -= video_first_pts
-
+            print("Video frame %s retrieved: %s" % (str(frame.index), str(frame)))
             frame_time = frame.time
             asyncio.run_coroutine_threadsafe(video_track._queue.put(frame), loop)
 
+            # Extract the keypoints from the frame
+            keypoints_generator = KeypointsGenerator()
+            try:
+                keypoints = keypoints_generator.get_keypoints(frame.to_rgb().to_ndarray())
+                keypoints_frame = KeypointsFrame(keypoints, frame.pts)
+                print("Keypoints for frame index %s retrieved." % str(frame.index))
+            except:
+                keypoints_frame = KeypointsFrame(bytes("Error!", encoding='utf8'), frame.pts)
+                print("Could not extract the keypoints for frame index %s" % str(frame.index))
```

Comment:
same as above

Reply:
please also make sure that any past print statements you added as part of this PR are all changed